changelog.txt

```diff
@@ -6,6 +6,9 @@ OPENSEADRAGON CHANGELOG
 * BREAKING CHANGE: Viewport.homeBounds, Viewport.contentSize, Viewport.contentAspectX and
     Viewport.contentAspectY have been removed. (#846)
 * BREAKING CHANGE: Overlay.scales, Overlay.bounds and Overlay.position have been removed. (#896)
+    * Overlay.scales can be replaced by Overlay.width !== null && Overlay.height !== null
```

Comment:
I guess the next logical question (sorry for being slow here) is whether it would be a good idea to maintain these properties for backwards compatibility? We could still deprecate them but also update them so they're available to be read.